azure/test.yml

```diff
@@ -24,33 +25,38 @@ jobs:
           vmImage: macOS-10.14
         Windows:
           vmImage: windows-2019
-  ${{ if ne('true', parameters.cross) }}:
-    variables:
+  variables:
+    ${{ if ne('true', parameters.cross) }}:
       vmImage: ubuntu-16.04
+    options: ''
   pool:
     vmImage: $(vmImage)
   steps:
   - template: _setup.yml
     parameters:
       rust: ${{ parameters.rust }}
       setup: ${{ parameters.setup }}
-  - ${{ if ne('true', parameters.single_threaded) }}:
-    - script: cargo test --all --features "${{ parameters.features }}"
-      displayName: Run tests
-      env:
-        ${{ insert }}: ${{ parameters.envs }}
-  - ${{ if eq('true', parameters.single_threaded) }}:
-    - script: cargo test --all --features "${{ parameters.features }}" -- --test-threads=1
-      displayName: Run tests (single-threaded)
+  # Cargo options go here
+  - script: echo '##vso[task.setvariable variable=options]$(options) --all'
```

Comment:
I think this needs to be `bash`, not `script`, to be cross-platform?

Reply:
Having looked at the failing tests I would agree, pwrsh seems to include the trailing " ' "